sdk/src/sys/vm.rs

```diff
@@ -6,23 +6,27 @@ pub use fvm_shared::sys::out::vm::MessageContext;
 super::fvm_syscalls! {
     module = "vm";
 
-    /// Abort execution with the given code and message. The code is recorded in the receipt, the
-    /// message is for debugging only.
+    /// Abort execution with the given code and optional message and data.
+    /// The code is recorded in the receipt, the message is for debugging only.
```

Comment:
The code and return value are recorded in the receipt.

Reply:
ack.

Reply:
fixed